README.md

```diff
@@ -6,43 +6,25 @@ This layer provides support for the NI Linux Real-Time distribution. The NI Linu
 This layer depends on:
```

Comment:
Could you just remove this whole section. The layer URIs aren't current either.

Reply:
Done.